### ISSUE TITLE
Docs for Serializer/Deserializer missing

### ISSUE THREAD

**Issue by dz902** [aws/sagemaker-python-sdk#1819]
**What did you find confusing? Please describe.**

I could not find docs about how to use one in the docs. Also since setting `content_type` is not possible in v2, we will have to use Serializers but there are no docs.

**Describe how documentation can be improved**

Add them.

**Additional context**
Add any other context or screenshots about the documentation request here.

**Comment by dz902** [comment]
For those who encounter a situation where only `content_type` setting is needed, do the following:

```
from sagemaker.serializers import IdentitySerializer

...

    PREDICTOR = Predictor(
        endpoint_name = ENDPOINT_NAME,
        serializer = IdentitySerializer(content_type = 'application/x-image')
    )
```

**Comment by bveeramani** [comment]
Fixed by #1824.
